src/index.projectionItem.test.ts

```diff
@@ -145,6 +145,24 @@ describe("projectionItem", () => {
 
       expect(query).toEqual("SELECT my_table.column01, column02 FROM my_table");
     });
+    it("should deal with alias", () => {
+      const query = rhombic
+        .parse("SELECT a FROM my_table")
+        .addProjectionItem("column02", { alias: "b" })
+        .toString();
+
+      expect(query).toEqual("SELECT a, column02 AS b FROM my_table");
+    });
+    it("should deal with alias that need to be escaped", () => {
+      const query = rhombic
+        .parse("SELECT a FROM my_table")
+        .addProjectionItem("column02", { alias: "this is second column" })
```

Comment:
Can we also add a test with an apostrophe? "it's a second column"?

Reply:
![image](https://user-images.githubusercontent.com/1761469/99061236-8702c480-25a1-11eb-8b71-ca7b0d49bd50.png)

Already tested in the escape function unit tests

Reply:
I don't have this context seeing this test though. Why not make this test more elaborated?

Reply:
I'm testing the code is using the correct branch depending of the `needToBeEscaped` result, not if the `needToBeEscaped` is working correctly (not the responsability of this code)